iidm/iidm-tck/src/test/java/com/powsybl/iidm/network/tck/AbstractGeneratorTest.java

```diff
@@ -210,7 +210,7 @@ public void testRemove() {
         int count = network.getGeneratorCount();
         Generator generator = network.getGenerator(TO_REMOVE);
         assertNotNull(generator);
-        generator.remove();
+        generator.remove();    // <---- TODO HERE
         assertNotNull(generator);
         Terminal terminal = generator.getTerminal();
```

Comment:
On network-store-store's side : generator.getTerminal() works but then terminal.isConnected() raise exception with message "Object has been removed in current variant" because the resource of the terminal instance is null